src/DebugApi.ts

```diff
@@ -201,8 +201,10 @@ export class DebugApi {
         const since = parseInt(query["since"] as string);
         const reason = query["reason"] as string;
         const dry = query["dryrun"] !== undefined && query["dryrun"] !== "false";
+        const defaultOnline = query["defaultOnline"] === "false"; // Default to true.
```

Comment:
useful comment but is there a better way of setting a default here? It took me an extra second to parse.

wouldn't `const defaultOnline = query["defaultOnline"] ?? "true";` be more readable?

Reply:
This must be a boolean, so how about:
```
const defaultOnline = (query["defaultOnline"] ?? "true") === "true";
```

Reply:
ok that works. looks clunkier but it's the "correct" way around